v8/rest/index.ts

```diff
@@ -507,6 +511,21 @@ export const Routes = {
 		return parts.join('/');
 	},
 
+	/**
+	 * Route for:
+	 * - PATCH `/webhooks/{webhook.id}/{webhook.token}/messages/{message.id}`
+	 * - DELETE `/webhooks/{webhook.id}/{webhook.token}/messages/{message.id}`
+	 * - PATCH `/webhooks/{webhook.id}/{webhook.token}/messages/@original`
+	 * - DELETE `/webhooks/{webhook.id}/{webhook.token}/messages/@original`
+	 *
+	 * - POST `/webhooks/{interaction.id}/{interaction.token}/messages`
+	 * - PATCH `/webhooks/{interaction.id}/{interaction.token}/messages/@original`
+	 * - PATCH `/webhooks/{interaction.id}/{interaction.token}/messages/{message.id}`
+	 */
+	webhookMessage(webhookID: string, webhookToken: string, messageID: string) {
```

Comment:
the description for the method says this can be used for
`POST /webhooks/{interaction.id}/{interaction.token}/messages`
but the last parameter (messageID) is not optional